releases_cmd_test.go

```diff
@@ -69,6 +69,12 @@ func TestReleasesCmdPrintVersin(t *testing.T) {
 			"public": {"index.html": "2", "docs.html": "2"}
 			}`,
 		},
+		{Version: "v2",
```

Comment:
Hi @pavanka, there are two entries for "v2": https://github.com/albertoleal/parse-cli/blob/PrintFirstVersionFound/releases_cmd_test.go#L66-L77